### ISSUE TITLE
migrated @live and @outlook.com accounts - will they work with EWS?

### ISSUE THREAD

**Issue by rolandschweiger** [Ericsson/exchangecalendar#513]
Thunderbird 45.2 (release, with its own Lightning Version)
EWS provider 3.80

Have a few MS accounts, one is @outlook.com and one @live.de and they have been migrated to the office365-infrastructure. There is hardly any information but apparently these accounts should work with EWS.
Would like to access the calendars and contacts in thunderbird.
I did try to use the EWS plugin and experimented with all kinds of settings, the autodiscover and the office365 autodiscover link. Does not work, it repeatingly asks for a password.
Do you know if this is an MS issue or EWS provider, or if outlook.com simply fails to work with EWS although on office365?
many thanks.
greetings,
Roland Schweiger

**Comment by designchris** [comment]
Hi Roland,
I'm struggling with the same problem now. However, I don't even get to the password prompt so far...

**Comment by rolandschweiger** [comment]
With the EWS ver 3.90 i at least  managed to log in (using the auto 
discovery) and can see the calendars of the MS account, aparantly i can 
also add / edit appointments but still not am 100% convinced that they 
sync properly.


Also i can see the contacts folder but editing contacts is not (or not 
yet) possible, unfortunately rendering the MS account useless for my 
purpose.


It is OT, but what i did was to migrate all calendars and all contacts 
to a google account, because there are 2 thunderbird plugins that work 
stable with google accounts: gContactSync for 2-way-contact-syncing and 
ProviderForGoogleCalendar gor google calendars.


Maybe in future the EWS will work with all calendars + all contacts of 
an MS account but so far there appear to be many problems.



Am 15.01.2017 um 18:20 schrieb designchris:

**Comment by designchris** [comment]
Thank you for reporting what's working/not working for you!

OT: My contacts are (still) in my google account for the same reason. And I'm also using gContactSync.

I really hope that the EWS provider will be usable together with outlook.com in the future.

**Comment by rolandschweiger** [comment]
yes so do i  and i long used MS eMail program "Windows Live Mail" 
w/hich /had its own Sync protocol (Delta-Sync) hat has been abandonned 
long time ago.

All a reason why i switched to Thunderbird, and moved calendars / 
contacts to a google account as currently as it is, this causes the 
least problems to sync between thunderbird and any type of Android 
Smartphone.

If one day EWS can give similar stability and functionality (i.e. 
2-way-sync of multiple calendars and 2-way-sync of contacts) then one 
could properly use an MS @outlook.com account across devices and platforms.


AFAIK MS sticks to the EAS protocol which is proprietary but EWS is a 
web-based plugin.

In contrast AFAIG google has an open API for the calendars *and* contacts.


The idea of Microsofts EAS is great as it can be used for all kinds of 
services - eMail (instead of IMAP it is easyier to sync folders), 
contacts, calendars etc. But developers seem to withdraw i suppose 
because of its costs.


Really a shame, MS should give similar functionality than Google here.


But i still have my @outlook.com account and will keep on testing with 
Thunderbird + EWS add-on.



Am 15.01.2017 um 19:45 schrieb designchris:
